src/components/TripCard.stories.js

```diff
@@ -5,4 +5,19 @@ export default {
   component: TripCard,
 };
 
-export const Default = () => <TripCard />;
```

Comment:
that should also work:

`export const Default = Template.bind({});
Default.args = {
    id: "1",
    place: "Barcelona",
    locations: "['Sagrada Familia', 'Park Güell']",
    isTripFuture: true,
  },
};

export const TripCardPast = Template.bind({});
TripCardPast.args = {
    ...Default.args.,
    isTripFuture: "false",
  },
};`

Reply:
Yup still working, thanks for the hint :)